src/window_background/loadPlayerConfig.ts

```diff
@@ -134,6 +135,9 @@ export async function loadPlayerConfig(): Promise<void> {
     const eventsList: InternalEvent[] = savedData.courses_index
       .filter((id: string) => savedData[id])
       .map((id: string) => {
+        if (isEpochTimestamp(savedData[id].date)) {
+          savedData[id].date = savedData[id].date * 1000;
+        }
```

Comment:
```suggestion
        if (isEpochTimestamp(savedData[id].date)) {
          savedData[id].date *= 1000;
        }
```

I think the `*=` operator exists in TS, too?